docs/src/components/Layout/FrameworkChooser.tsx

```diff
@@ -22,7 +27,12 @@ const FrameworkLink = ({ framework, onClick }: FrameworkLinkProps) => {
 
   return (
     <Link href={href} passHref>
-      <Button as="a" size="small" className={classNames} onClick={onClick}>
+      <Button
+        size="small"
+        className={classNames}
+        onClick={onClick}
+        isDisabled={isDisabled}
```

Comment:
out of scope for this PR, but maybe our disabled styled button should look more disabled. They look pretty similar to the normal button.